compute/elastic-metal/how-to/use-rescue-mode.mdx

```diff
@@ -56,6 +56,14 @@ Rescue mode allows you to boot a [Elastic Metal server](https://www.scaleway.com
   ssh root@<elastic-metal-server>
   ```
 
+  <Message type="tip">
+
+  Rescue mode uses the SSH keys registered for your Elastic Metal server as authentication method. In case you have lost your private key, you can access your server using the credentials displayed in the console:
```

Comment:
```suggestion
  Rescue mode uses the SSH keys registered for your Elastic Metal server as an authentication method. In case you have lost your private key, you can access your server using the credentials displayed in the console:
```